lib/editor/components/EditorInput.js

```diff
@@ -1,6 +1,6 @@
 // @flow
 
-import React, {Component} from 'react'
+import * as React from 'react'
```

Comment:
Can you use the previous import nomenclature? (unless others want to override me).

Reply:
No, because referencing React.Node requires this import approach.